pkg/apis/authentication/types.go

```diff
@@ -122,7 +122,7 @@ type TokenRequest struct {
 // TokenRequestSpec contains client provided parameters of a token request.
 type TokenRequestSpec struct {
 	// Audiences are the intendend audiences of the token. A recipient of a
-	// token must identitfy themself with an identifier in the list of
```

Comment:
this also exists in the external type and in the generated docs... can we fix it globally?